crates/interfaces/src/db.rs

```diff
@@ -31,4 +31,29 @@ pub enum DatabaseError {
     /// Failed to get database stats.
     #[error("Database stats error code: {0:?}")]
     Stats(i32),
+    /// Failed to use the specified log level, as it's not available.
+    #[error("Log level is not available: {0:?}")]
+    LogLevelUnavailable(LogLevel),
+}
+
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+#[cfg_attr(feature = "clap", derive(clap::ValueEnum))]
+/// Database log level.
+pub enum LogLevel {
```

Comment:
What shoudl be the default?

Reply:
`Fatal` is the default set by MDBX itself:
https://github.com/paradigmxyz/reth/blob/ce50db0b1d65f959b11d064157dd56696e282d30/crates/storage/libmdbx-rs/mdbx-sys/libmdbx/mdbx.c#L7247

I think it doesn't make sense for us to always call `mdbx_setup_debug` even if the log level doesn't change from `Fatal`.